src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -1566,6 +1566,7 @@ extern "C" {
 
     // Added in `OpenBSD` 5.5
     pub fn explicit_bzero(s: *mut ::c_void, len: ::size_t);
+    pub fn setproctitle(fmt: *const ::c_char, ...);
```

Comment:
mostly cosmetic remark: the position of the function addition is a bit misleading as it seems it is part of a block "Added in `OpenBSD` 5.5" (which isn't right for setproctitle).

starting a new block with a newline might be preferable